e2e/jest.test-setup.ts

```diff
@@ -14,9 +12,9 @@ const isDebugMode = process.argv.includes('--debug');
  */
 beforeEach(async () => {
   await page.setUserAgent(userAgent);
-  // See https://github.com/puppeteer/puppeteer/blob/master/docs/api.md#pagesetdefaultnavigationtimeouttimeout
-  page.setDefaultNavigationTimeout(navTimeOut);
-  page.setDefaultTimeout(timeOut);
+  await page.setViewport({width: 1280, height: 680});
```

Comment:
Thanks for adding this  - it's probably worth removing from the handful of tests where I've explicitly called it as well.